### PR TITLE
Remove capabilities/ccip and integration-tests/deployment

### PR DESCRIPTION
**Pull request by asoliman92**
All development should move to Chainlink repo. To do so make sure you have ccip repo as a remote to your git.

1. cd chainlink/
2. `git remote add ccipr org-25111032@github.com:smartcontractkit/ccip.git`
3. `git fetch ccipr`
4. `git checkout -b new-branch`
5. `git cherry-pick old-feature-branch~X..old-feature-branch` #These are the first and last commit from your currently open PR in ccip.

### CODE REVIEW

File: core/services/job/helpers_test.go [hunk 1]
```diff
@@ -177,24 +177,6 @@ func makeOCRJobSpec(t *testing.T, transmitterAddress common.Address, b1, b2 stri
 	return &dbSpec
 }
 
-// `require.Equal` currently has broken handling of `time.Time` values, so we have
-// to do equality comparisons of these structs manually.
-//
-// https://github.com/stretchr/testify/issues/984
-func compareOCRJobSpecs(t *testing.T, expected, actual job.Job) {
```
Comment (mateusz-sekara) [inline core/services/job/helpers_test.go]: WHy remove that?

File: core/services/job/job_orm_test.go [hunk 2]
```diff
@@ -1,2137 +0,0 @@
...
-	"github.com/smartcontractkit/chainlink/v2/core/services/pipeline"
-	"github.com/smartcontractkit/chainlink/v2/core/services/relay"
-	evmrelay "github.com/smartcontractkit/chainlink/v2/core/services/relay/evm"
-	"github.com/smartcontractkit/chainlink/v2/core/services/vrf/vrfcommon"
-	"github.com/smartcontractkit/chainlink/v2/core/services/webhook"
-	"github.com/smartcontractkit/chainlink/v2/core/testdata/testspecs"
-	"github.com/smartcontractkit/chainlink/v2/core/utils/testutils/heavyweight"
-)
-
-const mercuryOracleTOML = `name = 'LINK / ETH | 0x0000000000000000000000000000000000000000000000000000000000000001 | verifier_proxy 0x0000000000000000000000000000000000000001'
```
Comment (mateusz-sekara) [inline core/services/job/job_orm_test.go]: Shouldn't that stay?

File: core/services/job/helpers_test.go (hunk 1 above)
Comment (asoliman92) [reply core/services/job/helpers_test.go]: linting, not used anymore after removing `deployment` as well.

File: core/services/job/job_orm_test.go (hunk 2 above)
Comment (asoliman92) [reply core/services/job/job_orm_test.go]: Yes. Thanks!

**Review by mateusz-sekara — APPROVED** [review]

**Review by AnieeG — APPROVED** [review]

**Pull request merged**